README.md

```diff
@@ -27,17 +27,31 @@ npm install vue vue-ravepayment --save
 ###### my-compnent.vue sample
 ```vue
 <template>
-   <Rave
-       :is-production="false"
+ <div>
+   <h1>Testing the vue plugin</h1>
+     <Rave
+       :is-production="true"
        style-class="paymentbtn"
        :email="email"
        :amount="amount"
        :reference="reference"
        :rave-key="raveKey"
        :callback="callback"
        :close="close"
+       :metadata="meta"
+       :subaccounts="subaccounts"
+       :redirectUrl="redirect"
+       :paymentPlan="plan"
+       :customerFirstname="fname"
+       :customerLastname="lname"
+       paymentOptions="card,barter,account,ussd"
+       hostedPayemt=1
+       customTitle="Testing title"
+       customLogo="https://www.freelogodesign.org/Content/img/logo-samples/bakary.png"
```

Comment:
`customLogo` is not compulsory so please kindly remove it

Reply:
@iamraphson yes it is not compulsory, this is just a sample for users. same for the other parameters that are not compulsory but documented in the readme.

Reply:
Hi @iamraphson bumping this up, thanks!

Reply:
@temiadesina no need for the default.

Reply:
@iamraphson i've taken it out. kindly merge

Reply:
Hi @iamraphson please bumping this up, thanks!